src/snoowrap.js

```diff
@@ -233,6 +233,32 @@ const snoowrap = class snoowrap {
       return requester;
     });
   }
+  static fromInstalledClientAuth ({
```

Comment:
I'm not sure of the best name for this method, but I feel like `fromInstalledClientAuth` could be confusing because someone might confuse it with authenticated "installed client" applications that act on behalf of a user. Maybe something like `fromApplicationOnlyAuth` would be better instead?

Reply:
Yeah `fromApplicationOnlyAuth` makes sense